### PR TITLE
Group checkbox

### PR DESCRIPTION
**Pull request by xkureck**
* added new application for item type - you can set it and edit it
* you can manage set of groups which will be presented to the user in vo application

please check also my broken english and the the labels in general

### CODE REVIEW

**Review by Vojtech-Sassmann — COMMENTED** [review]
I think this should work pretty well, I have just one comment. I also like the refactorization.

File: apps/admin-gui/src/app/vos/components/application-form-manage-groups/application-form-manage-groups.component.ts [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    this.pageSize = event.pageSize;
+    this.tableConfigService.setTablePageSize(this.tableId, event.pageSize);
+  }
+
+  private setAuthRights() {
+    const vo = {id: this.voId, beanName: 'Vo'};
+    this.addAuth = this.authResolver.isAuthorized('addGroupsToAutoRegistration_List<Group>_policy', [vo]);
+
+    if(this.groups.length !== 0){
+      this.removeAuth = this.authResolver.isAuthorized('deleteGroupsFromAutoRegistration_List<Group>_policy', [vo, this.groups[0]]);
```
[the remark this answers is not in the document]
Comment (xkureck) [reply apps/admin-gui/src/app/vos/components/application-form-manage-groups/application-form-manage-groups.component.ts]: we already have implemented some logic for that, I just forgot its here, now its repaired

**Review by Vojtech-Sassmann — CHANGES_REQUESTED** [review]
I think this works well. Just it might be worth to adjust some of the texts.

File: apps/admin-gui/src/assets/i18n/en.json [hunk 2]
```diff
@@ -379,14 +381,25 @@
...
           "SWITCH_ENGLISH": "Switch to English",
           "SWITCH_CZECH": "Switch to Czech",
           "SWITCH_EXTENSION": "Switch to Extension",
           "SWITCH_INITIAL": "Switch to Initial"
+        },
+        "MANAGE_GROUPS_PAGE": {
+          "TITLE": "Manage groups for registration",
+          "ADD": "Add",
+          "REMOVE": "Remove",
+          "HELP": "On this page, you can set up a set of groups that will be displayed to users on an application to the organization. The user can select any set of these groups and by that make a new applications to these groups.",
```
[the remark this answers is not in the document]
Comment (xkureck) [reply apps/admin-gui/src/assets/i18n/en.json]: thanks for helping me with the labels

**Review by Vojtech-Sassmann — APPROVED** [review]

**Pull request merged**